cmd/node/main.go

```diff
@@ -949,8 +955,11 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	log.Info("application is now running")
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
-	<-sigs
-	log.Info("terminating at user's signal...")
+	select {
+	case <-sigs:
+		log.Info("terminating at user's signal...")
+	case <-chanStopNodeProcess:
```

Comment:
added a log which tells about terminating at internal stop signal, because I injected that channel inside the node.go and we can use it in more places.